packages/video_player/video_player_web/README.md

```diff
@@ -5,32 +5,58 @@ The web implementation of [`video_player`][1].
 ## Usage
 
 This package is [endorsed](https://flutter.dev/docs/development/packages-and-plugins/developing-packages#endorsed-federated-plugin),
-which means you can simply use `video_player`
-normally. This package will be automatically included in your app when you do.
+which means you can simply use `video_player` normally. This package will be
+automatically included in your app when you do.
 
 ## dart:io
```

Comment:
Do you want to maybe make a new `## Limitations` header, and downgrade this through "Mixing audio ..." to `###`s? This seems a like a lot of top-level sections.

Reply:
Yes, good call; I saw lots of level `##` headings, but didn't think of grouping them together. I'll get that done and land this.